### ISSUE TITLE
Learning from Synthetic Humans

### ISSUE THREAD

**Issue by Arata2887** [Arata2887/issue_copy#29]
### 0.どんな論文? /What does this work do?

- https://arxiv.org/pdf/1701.01370v3
- CVPR2018
- Introduces SURREAL (Synthetic hUmans foR REAL tasks), a large-scale dataset with synthetically generated but realistic images of people, which also includ ground truth depth maps, optical flow, surface normals, human part segmentations and joint locations (both 2D and 3D).

### 1. 論文のモチベーション / Motivation(s) of the paper / document

- Estimating human pose, shape, and motion from images and videos is challenging due to the need for large amounts of manually-labeled data, which is time-consuming and difficult to obtain. 
- Meanwhile, manual labeling of 3D pose, depth, and motion is impractical. 
- This paper introduces SURREAL to overcome these challenges by providing realistic and varied synthetic images along with detailed ground truth annotations. This allows for accurate training of CNNs for various human analysis tasks.

### 2. 論文のキーアイデア / Key idea(s) of the paper / document

**SURREAL Dataset**

> ![Fig 2](https://github.com/Arata2887/issue_copy/assets/119403000/702f8ffa-3a9f-4802-a64d-6edf3efe751e)

- **Synthetic Data Generation:** Utilizes the SMPL body model and MoSh method to create realistic 3D human body shapes and poses from motion capture data.

- **Random Sampling:** Generates a diverse set of images by randomly sampling viewpoints, clothing, lighting, and backgrounds.

- **Comprehensive Annotations:** Provides ground truth data including 2D/3D poses, depth maps, optical flow, surface normals, and body part segmentations.

- **Pipeline:** The data generation pipeline involves posing the 3D model, rendering the image with varied conditions, and generating the corresponding ground truth annotations.

### 3. その他（実験結果、関連研究、分からなかったところ 等）/ Misc (Experimental results, Related papers, Questions)

**Quantitative results:** **Table 2** and **Table 3** show the performance of models trained with different data (real, synthetic, and combined) on segmentation tasks. These comparisons highlight the significant role of synthetic data (SURREAL）in enhancing model accuracy.

> <img width="410" alt="Table 2" src="https://github.com/Arata2887/issue_copy/assets/119403000/74779664-4f30-44ff-bb52-5d74531f137e">

> <img width="411" alt="Table 3" src="https://github.com/Arata2887/issue_copy/assets/119403000/1667f2f9-1f4e-422d-ad77-872dcdc293bb">

**Qualitative results:** **Figure 6** shows parts segmentation results on Human3.6M, comparing models trained on real, synthetic (SURREAL), and fine-tuned data. **Figure 7** shows depth segmentation results using the same partitions.

> <img width="860" alt="Fig 6" src="https://github.com/Arata2887/issue_copy/assets/119403000/b6040826-bf9d-45f3-9b4c-4a97f7074374">

> <img width="863" alt="Fig 7" src="https://github.com/Arata2887/issue_copy/assets/119403000/100659e2-c14e-477b-ae16-f1f80679005e">

### 4.公開コードやデータセットは? /Public codes or datasets?

- https://www.di.ens.fr/willow/research/surreal/
